include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,550 @@
+    //VkBool32           descriptorBindingSampledImageUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageImageUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingUniformTexelBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageTexelBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingUpdateUnusedWhilePending;
+    //VkBool32           descriptorBindingPartiallyBound;
+    //VkBool32           descriptorBindingVariableDescriptorCount;
+    //VkBool32           runtimeDescriptorArray;
+    
+    //VkBool32           samplerFilterMinmax;   // ALIAS: VK_EXT_sampler_filter_minmax
+    
+    //VkBool32           scalarBlockLayout;     // or VK_EXT_scalar_block_layout
+    
+    //VkBool32           imagelessFramebuffer;  // or VK_KHR_imageless_framebuffer
+    
+    //VkBool32           uniformBufferStandardLayout;   // or VK_KHR_uniform_buffer_standard_layout
+    
+    //VkBool32           shaderSubgroupExtendedTypes;   // or VK_KHR_shader_subgroup_extended_types
+    
+    //VkBool32           separateDepthStencilLayouts;   // or VK_KHR_separate_depth_stencil_layouts
+    
+    //VkBool32           hostQueryReset;                // or VK_EXT_host_query_reset
+    
+    //VkBool32           timelineSemaphore;             // or VK_KHR_timeline_semaphore
+    
+    // or VK_KHR_timeline_semaphore:
+    bool bufferDeviceAddress = false;
+    //VkBool32           bufferDeviceAddressCaptureReplay;
+    //VkBool32           bufferDeviceAddressMultiDevice;
+    
+    // or VK_KHR_vulkan_memory_model
+    //VkBool32           vulkanMemoryModel;
+    //VkBool32           vulkanMemoryModelDeviceScope;
+    //VkBool32           vulkanMemoryModelAvailabilityVisibilityChains;
+    
+    //VkBool32           shaderOutputViewportIndex;     // ALIAS: VK_EXT_shader_viewport_index_layer
+    //VkBool32           shaderOutputLayer;             // ALIAS: VK_EXT_shader_viewport_index_layer
+    //VkBool32           subgroupBroadcastDynamicId;    // if Vulkan 1.2 is supported
+
+    /* Vulkan 1.3 Core */
+
+    //VkBool32           robustImageAccess;                 //  or VK_EXT_image_robustness
+    
+    //  or VK_EXT_inline_uniform_block:
+    //VkBool32           inlineUniformBlock;
+    //VkBool32           descriptorBindingInlineUniformBlockUpdateAfterBind;
+    
+    //VkBool32           pipelineCreationCacheControl;      // or VK_EXT_pipeline_creation_cache_control
+    //VkBool32           privateData;                       // or VK_EXT_private_data
+    //VkBool32           shaderDemoteToHelperInvocation;    // or VK_EXT_shader_demote_to_helper_invocation
+    //VkBool32           shaderTerminateInvocation;         // or VK_KHR_shader_terminate_invocation
+    
+    // or VK_EXT_subgroup_size_control
+    //VkBool32           subgroupSizeControl;
+    //VkBool32           computeFullSubgroups;
```

Comment:
Related to https://github.com/Devsh-Graphics-Programming/Nabla/pull/357#discussion_r888485179
[VkPhysicalDeviceSubgroupSizeControlFeaturesEXT](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceSubgroupSizeControlFeaturesEXT.html)
[VkPhysicalDeviceSubgroupSizeControlPropertiesEXT](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceSubgroupSizeControlPropertiesEXT.html)

Reply:
Should we expose and If yes, what is the impl for GL?

Reply:
Please expose.

In https://github.com/Devsh-Graphics-Programming/Nabla/pull/357#discussion_r888485179 I wrote
> requiredSubgroupSizeStages
> 0 because no control in GL for any stage

In GL there's no way to control this stuff.